ui/app/AppLayouts/Chat/CommunityComponents/CreateCategoryPopup.qml

```diff
@@ -48,166 +48,173 @@ StatusModal {
             //% "New category"
             qsTrId("new-category")
 
-    content: ScrollView {
-        id: scrollView
-
+    content: Column {
+                
         width: popup.width
-        height: Math.min(content.height, 432)
-
-        property ScrollBar vScrollBar: ScrollBar.vertical
         property alias categoryName: nameInput
 
-        contentHeight: content.height
-        ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
-        clip: true
+        Item {
+            width: parent.width
+            height: 76
+            Input {
+                id: nameInput
+                width: parent.width -32
+
+                anchors.centerIn: parent
+                anchors.left: undefined
```

Comment:
interesting... why setting these to `undefined`?

Reply:
This unfortunately is a rather big flaw in the API of `Input` (hopefully fixed soon once we move to `StatusInput` provided by StatusQ).

`Input` comes with `anchors.left: parent.left` and `anchors.right: parent.right` by default. In this context however, I need to set a custom width (L61) but QML doesn't allow me to because the component is already anchored left in right (taking the entire width).

So basically, I need to **undo** default behaviour to make it work for me. Again, this will be fixed when we switch to the (not yet existing) `StatusInput` which won't rely on `parent` by itself.

Hope this makes sense :D

Also, should add that that particular change is unrelated to this PR